space-plugins/nuxt-base/server/middleware/02.auth.global.ts

```diff
@@ -13,6 +13,18 @@ export default defineEventHandler(async (event) => {
 		return;
 	}
 
+	// Delete cookie and initiated OAuth flow
+	// if the user hasn't been authenticated yet.
+	// (Storyfront attaches this query parameter in that case)
+	if (getQuery(event)['init_oauth'] === 'true') {
+		setCookie(event, 'sb.auth', '', {
```

Comment:
nit: I wonder If we could have the `sb.auth` string somehow imported from the `app-extensions-auth` so we don't need to hardcode it here. Not a big deal, though.